spec/directives/if_else/escaped.hrx

```diff
@@ -0,0 +1,24 @@
+<===> options.yml
```

Comment:
Rather than making a new `if_else` directory, I'd probably just test `@else` within the `if` directory (since it's inherently tightly coupled).